kong/api/routes/kong.lua

```diff
@@ -11,6 +13,7 @@ local tonumber = tonumber
 local kong = kong
 local knode  = (kong and kong.node) and kong.node or
                require "kong.pdk.node".new()
+local errors = Errors.new()
```

Comment:
`Errors / errors` concerns me but I don't know what our linting rules / guidelines are. I'm guessing it's not possible to `require "kong.db.errors".new()`

Reply:
In most of the code I think we refer to it as `err_t`